src/main/resources/sql/populate.sql

```diff
@@ -0,0 +1,25 @@
+INSERT INTO
+    customers (name, surname, age)
+VALUES
+    ('Alexandr', 'Nevskiy', 35)
+    ,('Name', 'Surname', 16)
+    ,('Irina', 'Ovechkina', 56)
+    ,('Petr', 'Petrov', 69)
+    ,('Ivan', 'Ivanov', 21)
+    ,('Olha', 'Polkovnikova', 21)
+    ,('IvanV', 'Ivanov', 22)
+    ,('IvanVV', 'Ivanov', 23)
+    ,('Antonina', 'Tishenko', 24);
```

Comment:
I think in case of the first task it's better to fill values with Boundary Value Analysis (BVA): 15,16,17,34,35,36 https://goo.gl/wQmkYn

Reply:
Thanks for your comment, but I have a boundary value for the test. It's 16 and 35. I think that's enough